modules/ROOT/pages/procedures.adoc

```diff
@@ -1116,7 +1116,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/current/p
 Starting with Neo4j 2025.04, the default database can also be set to a local or remote database alias.
 ====
 
-[role=label--enterprise-edition label--admin-only label--deprecated-2025.01]
+[role=label--enterprise-edition label--admin-only label--deprecated-2025.01 label--removed-cypher-25]
```

Comment:
@JPryce-Aklundh, in this case should we change `label--deprecated-2025.01` to `label--deprecated-Cypher-5`?

Reply:
We discussed this situation with Reneta and decided to keep 'Deprecated in a specific Neo4j version' labels for now. Later we can revisit this and replace the labels with 'Deprecated in Cypher 5' or add 'Deprecated in Cypher 5' to the existing ones.